client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -177,27 +174,15 @@ class Security2faSMSSettings extends Component {
 					<FormButtonsBar className="security-2fa-sms-settings__buttons">
 						<FormButton
 							disabled={ this.getSubmitDisabled() }
-							onClick={ ( event ) => {
-								gaRecordEvent( 'Me', 'Clicked On 2fa Use App Button' );
-								this.onVerifyByApp( event );
-							} }
-						>
-							{ this.props.isUpdatingUserSettings
-								? savingLabel
-								: this.props.translate( 'Verify via App' ) }
-						</FormButton>
-
-						<FormButton
-							disabled={ this.getSubmitDisabled() }
-							isPrimary={ false }
+							isPrimary={ true }
```

Comment:
Can be shortened to:

```suggestion
							isPrimary
```